website/docs/gen-ai/inference/Neuron/vLLM-rayserve.md

```diff
@@ -0,0 +1,565 @@
+---
+title: RayServe with vLLM
```

Comment:
Let's keep the same convention as others focussing on model and accelerator.

```suggestion
title: Llama3 on Inferentia2
```

Reply:
Hi Apoorva, Vara recommended me to change the title to "RayServe with vLLM".

Reply:
I have updated and aligned the names properly